en/micro-integrator/docs/administer-and-observe/working-with-management-api.md

```diff
@@ -74,10 +74,130 @@ Invoke the `/logout` resource to revoke the JWT token you used for [invoking the
 curl -X GET "https://localhost:9164/management/logout” -H "accept: application/json" -H "Authorization: Bearer %AccessToken%”
 ```
 
-## Accessig API resources
+## Accessing API resources
 
 The management API has multiple resources to provide information regarding the deployed artifacts as well as the server itself.
 
+### GET USERS
+
+-	**Resource**: `/users`
+
+	**Description**: Retrieves a list of all users stored in an [external user store](../../../setup/user_stores/setting_up_ro_ldap).
+
+	**Example**:
+
+  	```bash tab='Request'
+  	curl -X GET "https://localhost:9164/management/users?pattern=”*us*”&role=”sole”" -H "accept: application/json" -H "Authorization: Bearer %AccessToken%" -k -i
+  	```
+
+    ```bash tab='Response'
+    {
+    	count:2
+    	list:
+    	[
+    		userId: user1,
+    		userId: user2,
+    		userId: user3,
+    	]
+    }
+    ```
+
+-	**Resource**: `/users/{user_id}`
+
+	**Description**: Retrieves information related to a specified user stored in the [external user store](../../../setup/user_stores/setting_up_ro_ldap).
+
+	**Example**:
+
+  	```bash tab='Request'
+  	curl -X GET "https://localhost:9164/management/users/user1" -H "accept: application/json" -H "Authorization: Bearer %AccessToken%" -k -i
+  	```
+
+    ```bash tab='Response'
+    {
+      userid: “user1”,
+      	isAdmin: true/false,
+      	roles :
+      [
+           role1,
```

Comment:
There's a small indentation issue. in line 117 as well.

Reply:
fixed